README.md

```diff
@@ -69,7 +69,7 @@ Build the Surefire project using **Maven 3.1.0+** and **JDK 1.8+**.
   ```
   
 
-### Deploying web site
```

Comment:
@elharo I have fixed the build. Pls remove the second commit `3a42b4e7aec0819c344f09c58e92d7a3d0a2bb6d`.